utils/e2e-tests/ts/run-tests

```diff
@@ -10,6 +10,4 @@ shift
 HUMANODE_PEER_PATH="$(realpath -s "$HUMANODE_PEER_PATH")"
 export HUMANODE_PEER_PATH
 
-if [[ "$TEST_SUITE" == "base" ]]; then
-  exec yarn utils/e2e-tests/ts test
-fi
+exec yarn utils/e2e-tests/ts test /"$TEST_SUITE"/
```

Comment:
Added slashes to apply tests filter by having a corresponding `test suite` directory in the test file path.

Reply:
This is rather odd... I mean, the first `/`. Do we tell the test command to do a lookup from the root here?

Reply:
I'm just trying to interpret the meaning and justification here. What is the issues if we don't have the slashes?

Reply:
Maybe, the first `/` is not required.

Let me explain the goal of this. I added the following structure for testing
```
- tests
  - base
    - hello.ts
    - world.ts
  - hello
    - base.ts
```

In case simple run of  `exec yarn utils/e2e-tests/ts test base`, the following tests are run: `base.ts`, `hello.ts`, `world.ts`.

So, we should add slash. Tested with both cases: `/"$TEST_SUITE"/` `$TEST_SUITE"/` - works.

Reply:
Just tested with my sample, it doesn't work. Tried `hello` test suit. it runs both `hello/base.ts` and `base/hello.ts`

Reply:
https://github.com/humanode-network/humanode/pull/1580/commits/a83e0b37a9dc5324e8162ef46a5cf7c0aabb357f

Reply:
@MOZGIII What do you think about current version ? is it ok ?

Reply:
> So, we essentially want to pass `/base/` there? Not `/base`, not `./base`, not `base/`, and not `./base/`?
> 
> What if we have `a/b/base/mytest.ts`?

Does it work for all the cases, even with a deeply nested dir with the same name?

Reply:
We can try using https://vitest.dev/config/#dir option

Reply:
Hm.. Added additional example like `tests/base/tests/hello/hello.ts` containing deeply `tests`.  It's taken as well in case `hello` test suite.